### PR TITLE
Ccp 1783

### PR DESCRIPTION
**Pull request by dqdquint**
Added redis caching for subscription product info. Testing information is in jira ticket CCP-1783

## Type of change

- [x] Bug fix
- [x] New feature
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the CHANGELOG.md

### CODE REVIEW

File: extension-config.json [hunk 1]
```diff
@@ -209,6 +209,33 @@
         "path": "/v1/shop/%(shopId)s/checkout_web_checkout_shopify?parsed=true",
         "key": "value.frontend.alias"
       }
+    },
+    "redisClientShopNumber": {
```
Comment (aaron-martin) [inline extension-config.json]: might not need this configuration. I think this is available from context.meta.appId

File: extension-config.json [hunk 2]
```diff
@@ -209,6 +209,33 @@
...
+      "type": "admin",
+      "destination": "backend",
+      "default": "",
+      "params": {
+        "label": "Shop number for Redis client configuration",
+        "type": "text"
+      }
+    },
+    "redisClientSecret": {
```
Comment (aaron-martin) [inline extension-config.json]: we might need to make this a bigApi configuration to restrict its access like it is here https://github.com/shopgate/ext-shopify-cart/blob/master/extension-config.json#L29

File: extension-config.json (hunk 1 above)
Comment (dqdquint) [reply extension-config.json]: Fixed

File: extension/package.json [hunk 3]
```diff
@@ -31,5 +31,10 @@
     "nyc": "^14.1.1",
     "sinon": "^4.1.2",
     "standard": "^10.0.3"
+  },
+  "dependencies": {
+    "bluebird": "^3.7.1",
```
[the remark this answers is not in the document]
Comment (dqdquint) [reply extension/package.json]: removed

File: extension-config.json (hunk 2 above)
Comment (dqdquint) [reply extension-config.json]: Made into configuration

**Pull request merged**